ui/packages/ui-lib/package.json

```diff
@@ -47,13 +47,15 @@
     "@storybook/addon-interactions": "^7.6.6",
     "@storybook/addon-links": "^7.6.6",
     "@storybook/blocks": "^7.6.6",
+    "@storybook/icons": "^1.2.9",
     "@storybook/react": "^7.6.6",
     "@storybook/react-vite": "^7.6.6",
     "@storybook/test": "^7.6.6",
     "@testing-library/jest-dom": "^6.1.4",
     "@testing-library/react": "^14.0.0",
     "eslint-config-prettier": "^9.0.0",
     "eslint-plugin-storybook": "^0.6.15",
+    "storybook-addon-pseudo-states": "^3.1.1",
```

Comment:
can we add this one to all our components where is it's needed? I could prepare a separate ticket for it

Reply:
Yes, if you want to use `hover` and `focus` pseudo classes in stories with other components, you should add this